tests/tracking/test_client.py

```diff
@@ -1806,3 +1806,25 @@ def test_block_create_prompt_with_existing_model_name(tracking_uri):
             description="A friendly greeting",
             tags={"model": "my-model"},
         )
+
+
+def test_log_and_detach_prompt(tracking_uri):
+    client = MlflowClient(tracking_uri=tracking_uri)
+
+    client.register_prompt(name="p1", template="Hi, there!")
+    client.register_prompt(name="p2", template="Hi, {{name}}!")
```

Comment:
```suggestion
    client.register_prompt(name="p1", template="Hi, there!")
    time.sleep(0.001)
    client.register_prompt(name="p2", template="Hi, {{name}}!")
```

can we insert a sleep to ensure the prompts have different creation timestamps for windows?